src/tests.rs

```diff
@@ -17,9 +17,19 @@ pub fn setup_tracing() {
 pub fn setup_cache() -> TempDir {
     let tempdir = TempDir::new("tor-cache").expect("create temp dir");
 
+    let authority = include_str!("test-data/authority.txt");
+    let certificate = include_str!("test-data/certificate.txt");
     let consensus = include_str!("test-data/consensus.txt");
     let microdescriptors = include_str!("test-data/microdescriptors.txt");
 
+    let authority_path = tempdir.path().join("authority.txt");
+    let mut authority_file = File::create(authority_path).expect("create temp authority file");
+    write!(authority_file, "{}", authority).expect("write temp authority file");
+
+    let certificate_path = tempdir.path().join("certificate.txt");
+    let mut certificate_file = File::create(certificate_path).expect("create temp certificate file");
+    write!(certificate_file, "{}", certificate).expect("write temp certificate file");
```

Comment:
as previously reviewed, you can use `std::fs::copy` to achieve the same effect

```suggestion
    copy(
        concat!(env!("CARGO_MANIFEST_DIR"), "/src/test-data/authority.txt"),
        tempdir.path().join("authority.txt"),
    )
    .expect("copy temp authority file");

    copy(
        concat!(env!("CARGO_MANIFEST_DIR"), "/src/test-data/certificate.txt"),
        tempdir.path().join("certificate.txt"),
    )
    .expect("copy temp certificate file");
```